static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -487,7 +487,9 @@ const SPECIAL_FIELDS: SpecialFields = {
               </StyledLink>
             </QuickContextHoverWrapper>
           ) : (
-            <OverflowFieldShortId shortId={`${data.issue}`} />
+            <StyledLink to={target} aria-label={issueID}>
+              <OverflowFieldShortId shortId={`${data.issue}`} />
```

Comment:
```suggestion
              <OverflowFieldShortId shortId={String(data.issue)} />
```

Reply:
@dashed can you approve? I don't want to sit around for CI to run again, I can update this tomorrow when I do the test lol

Reply:
Okies.